Changes

```diff
@@ -97,7 +97,8 @@ Working version
 - #13388, #13540: raises an error message (and not an internal compiler error)
   when two local substitutions are incompatible (for instance `module type
   S:=sig end type t:=(module S)`)
-  (Florian Angeletti, report by Nailen Matschke, review by Gabriel Scherer)
+  (Florian Angeletti, report by Nailen Matschke, review by Gabriel Scherer, and
```

Comment:
I would remove the oxford comma, we don't do usually that in the Changes file.

Reply:
The use of the oxford comma varies quite a lot within the `Changes` file.